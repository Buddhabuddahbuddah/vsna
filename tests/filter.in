command> max_verbosity 00111111
command> min_verbosity 00111111
command> exclude *
command> include meta*
command> include fails
command> include passes
command> include skips
command> run
test assert_bools skipped.
test assert_ints skipped.
test assert_strings skipped.
test overall skipped.
firmware.ino:320:1: fail assert (test_passes_instance.state=1) >= (Test::DONE_SKIP=2)
firmware.ino:326:1: pass assert (test_passes_instance.state=1) < (Test::DONE_SKIP=2)
firmware.ino:332:1: fail assert (test_passes_instance.state=1) == (Test::DONE_PASS=3)
firmware.ino:338:1: pass assert (test_passes_instance.state=1) != (Test::DONE_PASS=3)
firmware.ino:344:1: fail assert (test_passes_instance.state=1) == (Test::DONE_FAIL=4)
firmware.ino:350:1: pass assert (test_passes_instance.state=1) != (Test::DONE_FAIL=4)
firmware.ino:356:1: fail assert (test_passes_instance.state=1) == (Test::DONE_SKIP=2)
firmware.ino:362:1: pass assert (test_passes_instance.state=1) != (Test::DONE_SKIP=2)
firmware.ino:321:1: fail assert (test_fails_instance.state=1) >= (Test::DONE_SKIP=2)
firmware.ino:327:1: pass assert (test_fails_instance.state=1) < (Test::DONE_SKIP=2)
firmware.ino:333:1: fail assert (test_fails_instance.state=1) == (Test::DONE_PASS=3)
firmware.ino:339:1: pass assert (test_fails_instance.state=1) != (Test::DONE_PASS=3)
firmware.ino:345:1: fail assert (test_fails_instance.state=1) == (Test::DONE_FAIL=4)
firmware.ino:351:1: pass assert (test_fails_instance.state=1) != (Test::DONE_FAIL=4)
firmware.ino:357:1: fail assert (test_fails_instance.state=1) == (Test::DONE_SKIP=2)
firmware.ino:363:1: pass assert (test_fails_instance.state=1) != (Test::DONE_SKIP=2)
firmware.ino:322:1: fail assert (test_skips_instance.state=1) >= (Test::DONE_SKIP=2)
firmware.ino:328:1: pass assert (test_skips_instance.state=1) < (Test::DONE_SKIP=2)
firmware.ino:334:1: fail assert (test_skips_instance.state=1) == (Test::DONE_PASS=3)
firmware.ino:340:1: pass assert (test_skips_instance.state=1) != (Test::DONE_PASS=3)
firmware.ino:346:1: fail assert (test_skips_instance.state=1) == (Test::DONE_FAIL=4)
firmware.ino:352:1: pass assert (test_skips_instance.state=1) != (Test::DONE_FAIL=4)
firmware.ino:358:1: fail assert (test_skips_instance.state=1) == (Test::DONE_SKIP=2)
firmware.ino:364:1: pass assert (test_skips_instance.state=1) != (Test::DONE_SKIP=2)
test passes passed.
test skips skipped.
test fails failed.
firmware.ino:320:1: pass assert (test_passes_instance.state=3) >= (Test::DONE_SKIP=2)
firmware.ino:326:1: fail assert (test_passes_instance.state=3) < (Test::DONE_SKIP=2)
firmware.ino:332:1: pass assert (test_passes_instance.state=3) == (Test::DONE_PASS=3)
firmware.ino:338:1: fail assert (test_passes_instance.state=3) != (Test::DONE_PASS=3)
firmware.ino:344:1: fail assert (test_passes_instance.state=3) == (Test::DONE_FAIL=4)
firmware.ino:350:1: pass assert (test_passes_instance.state=3) != (Test::DONE_FAIL=4)
firmware.ino:356:1: fail assert (test_passes_instance.state=3) == (Test::DONE_SKIP=2)
firmware.ino:362:1: pass assert (test_passes_instance.state=3) != (Test::DONE_SKIP=2)
firmware.ino:321:1: pass assert (test_fails_instance.state=4) >= (Test::DONE_SKIP=2)
firmware.ino:327:1: fail assert (test_fails_instance.state=4) < (Test::DONE_SKIP=2)
firmware.ino:333:1: fail assert (test_fails_instance.state=4) == (Test::DONE_PASS=3)
firmware.ino:339:1: pass assert (test_fails_instance.state=4) != (Test::DONE_PASS=3)
firmware.ino:345:1: pass assert (test_fails_instance.state=4) == (Test::DONE_FAIL=4)
firmware.ino:351:1: fail assert (test_fails_instance.state=4) != (Test::DONE_FAIL=4)
firmware.ino:357:1: fail assert (test_fails_instance.state=4) == (Test::DONE_SKIP=2)
firmware.ino:363:1: pass assert (test_fails_instance.state=4) != (Test::DONE_SKIP=2)
firmware.ino:322:1: pass assert (test_skips_instance.state=2) >= (Test::DONE_SKIP=2)
firmware.ino:328:1: fail assert (test_skips_instance.state=2) < (Test::DONE_SKIP=2)
firmware.ino:334:1: fail assert (test_skips_instance.state=2) == (Test::DONE_PASS=3)
firmware.ino:340:1: pass assert (test_skips_instance.state=2) != (Test::DONE_PASS=3)
firmware.ino:346:1: fail assert (test_skips_instance.state=2) == (Test::DONE_FAIL=4)
firmware.ino:352:1: pass assert (test_skips_instance.state=2) != (Test::DONE_FAIL=4)
firmware.ino:358:1: pass assert (test_skips_instance.state=2) == (Test::DONE_SKIP=2)
firmware.ino:364:1: fail assert (test_skips_instance.state=2) != (Test::DONE_SKIP=2)
test meta passed.
test summary: 2 passed, 1 failed, and 5 skipped, out of 8 test(s).
