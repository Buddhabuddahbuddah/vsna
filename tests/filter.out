command> max_verbosity 00111111
command> min_verbosity 00111111
command> exclude *
command> include meta*
command> include fails
command> include passes
command> include skips
command> run
test assert_bools skipped.
test assert_ints skipped.
test assert_strings skipped.
test overall skipped.
arduino_unit_test.ino:302:1: fail assert (test_passes_instance.state=1) >= (Test::DONE_SKIP=2)
arduino_unit_test.ino:308:1: pass assert (test_passes_instance.state=1) < (Test::DONE_SKIP=2)
arduino_unit_test.ino:314:1: fail assert (test_passes_instance.state=1) == (Test::DONE_PASS=3)
arduino_unit_test.ino:320:1: pass assert (test_passes_instance.state=1) != (Test::DONE_PASS=3)
arduino_unit_test.ino:326:1: fail assert (test_passes_instance.state=1) == (Test::DONE_FAIL=4)
arduino_unit_test.ino:332:1: pass assert (test_passes_instance.state=1) != (Test::DONE_FAIL=4)
arduino_unit_test.ino:338:1: fail asserunit_test.ino:309:1: pass assert (test_fails_instance.state=1) < (Test::DONE_SKIP=2)
arduino_unit_test.ino:315:1: fail assert (test_fails_instance.stateno_unit_test.ino:327:1: fail assert (test_fails_instance.state=1) == (Test::DONE_FAIL=4)
arduino_unit_test.ino:333:1: pass assert_unit_test.ino:327:1: fail assert (test_fails_instance.state=1) == (Test::DONE_FAIL=4)
arduino_unit_test.ino:333:1: pass assert (test_fails_instance.state=1) != (Test::DONE_FAIL=4)
arduino_unit_test.ino:339:1: fail no_unit_test.ino:310:1: pass assert (test_skips_instance.state=1) < (Test::DONE_SKIP=2)
arduino_unit_test.ino:316:1: fail assert (test_skips_instance.state=1) == (Test::DONE_PASS=3)
arduino_unit_teDo:3_FAIL=4)
arduino_unit_test.ino:340:1: fail assert (test_skips_instance.state=1) == (Test::DONE_SKIP=2)
arduino_unit_test.ino:346:1: pass assert (test_skips_instance.state=1) != (Test::DONE_SKIP=2)
arduino_unit_test.ino:302:1fail assert (test_passes_instance.state=1) == (Test::Des_instance.state=1) != (Test::DONE_PASS=3)
arduino_unit_test.ino:326:1: fail assert (test_passes_instance.state=1) == (Test::DONE_FAIL=4)
arduino_unit_test.ino:332:1: pass assert (test_passes_instance.state=1) != (Test::Ds_instance.state=1) >= (Test::DONE_SKIP=2)
arduino_unit_test.ino:309:1: pass assert (test_fails_instance.state=1) < (Test::DONE_SKIP=2)
arduino_unit_test.ino:315:1: ftate=1) != (Test::DONE_PASS=3)
arduino_unit_test.ino:327:1: fail assert (test_fails_o:321:1: pass assert (test_fails_instance.state=1) != (Test::DONE_PASS=3)
arduino_unit_test.ino:327:1: fail assert (test_fails_instance.state=1) == (Test::DONE_FAIL=4)
arduino_unit_test.ino:333:1: pass assert (test.ino:304:1: fail assert (test_skips_instance.state=1) >= (Test::DONE_SKIP=2)
arduino_unit_test.ino:310:1: pass assert (test_skips_instance.state=1) < (Test::DONE_SKIP=2)
arduino_unest.ino:340:1: fail assert (test_skips_instance.state=1) == (Test::DONE_SKIP=2)
arduino) != (Test::DONE_FAIL=4)
arduino_unit_test.ino:340:1: fail assert (test_skips_instance.state=1) == (Test::DONE_SKIP=2)
arduino_unit_test.ino:346:1: pass assert (test_skips_instance.state=1) != (Test::DONE_SKIP=2)
test passes passed.
t:1: pass assert (test_passes_instance.state=3) == (Test::DONE_PASS=3)
arduino_unit_test.ino:320:1: fail assert (test_passes_instance.state=3) != (Test::ses_instance.state=3) == (Test::DONE_SKIP=2)
arduino_unit_test.ino:344:1: pass assert (test_passes_instance.state=3) != (Test::DONE_SKIP=2)
arduino_unit_test.ino:303:1: pass assert (test_fails_instan
arduino_unit_test.ino:315:1: fail assert (tnit_test.ino:309:1: fail assert (test_fails_instance.state=4) < (Test::DONE_SKIP=2)
arduino_unit_test.ino:315:1: fail assert (test_fails_instance.state=4) == (Test::DONE_PASS=3)
arduino_unit_test.ino:321:1: pass assert _fails_instance.state=4) != (Test::DONE_SKIP=2)
arduino_unit(Test::DONE_SKIP=2)
arduino_unit_test.ino:345:1: pass assert (test_fails_instance.state=4) != (Test::DONE_SKIP=2)
arduino_unit_test.ino:304:1: pass assert (test_skips_instance.state=2) >= (Test::DONE_SKIP=2)
arduin_test.ino:334:1: pass rduino_unit_test.ino:328:1: fail assert (test_skips_instance.state=2) == (Test::DONE_FAIL=4)
arduino_unit_test.ino:334:1: pass assert (test_skips_instance.state=2) != (Test::DONE_FAIL=4)
arduino_unit_test.ino:340:1: pass a2) != (Test::DONE_SKIP=2)
test meta passed.
test summary: 2 passed, 1 failed, and 5 skipped, out of 8 test(s).
